tests/settings.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+HELPER_SETTINGS = {
+    'INSTALLED_APPS': [],
```

Comment:
![](images/composer-emoji/apple/1f44d.png)  
  

  
On Aug 17 2016, at 6:22 pm, Angelo Dini <notifications@github.com> wrote:  

> In [tests/settings.py](https://github.com/divio/djangocms-
video/pull/20#discussion_r75156147):

>  
>  
>     > @@ -0,0 +1,21 @@

>     > +#!/usr/bin/env python

>     > +# -*- coding: utf-8 -*-

>     > +

>     > +HELPER_SETTINGS = {

>     > +    'INSTALLED_APPS': [],

>  
>

> not for the application itself (djangocms-video) only for dependencies or so

>

> --  
You are receiving this because you commented.  
Reply to this email directly, [view it on GitHub](https://github.com/divio
/djangocms-
video/pull/20/files/e0255b8a2d4f8ea92ec67a16be0ee06a2d797eec#r75156147), or
[mute the thread](https://github.com/notifications/unsubscribe-auth/ACk0SKWpse
rlam5wQJPE23_OgAxRYU7dks5qgzU6gaJpZM4Jh_nq).![](https://github.com/notificatio
ns/beacon/ACk0SGbz6Q3rDxTm1DoRJAqTT5-OUtMYks5qgzU6gaJpZM4Jh_nq.gif)